src/http/websocket_http_client.zig

```diff
@@ -96,7 +101,7 @@ fn buildRequestBody(
             "{any}" ++
             "{any}" ++
             "\r\n",
-        .{ pathname_, host_, pico_headers, extra_headers },
+        .{ pathname_.slice(), host_.slice(), pico_headers, extra_headers },
```

Comment:
the previous version of this would be the same btw

the formatter converts it to UTF-8.

Reply:
If you want to keep this change, it should be `{s}` instead of `{any}`. `{any}` is unspecified.

Reply:
Previous version would segfault